solr/webapp/web/js/angular/controllers/collections.js

```diff
@@ -23,6 +23,18 @@ solrAdminApp.controller('CollectionsController',
 
           $scope.rootUrl = Constants.ROOT_URL + "#/~collections/" + $routeParams.collection;
 
+          Zookeeper.liveNodes({}, function(data) {
+            $scope.nodes = [];
+            var children = data.tree[0].children;
+            for (var child in children) {
+              $scope.nodes.push(children[child].text);
+            }
+            $scope.availableNodeSet = [];
+            for (var i in $scope.nodes) {
```

Comment:
Do we need to push this to `$scope.nodes` first? Seems like could just push `children[child].text to `$scope.availableNodeSet`

Reply:
Fixed!